L1Trigger/DTTriggerServerPhi/src/DTTSCand.cc

```diff
@@ -241,25 +241,24 @@ void DTTSCand::setBitsTsm() {
         std::cout << "DTTSCand::DTTSCand(): masking not correct: ";
         std::cout << config()->TssMasking(itk);
         std::cout << " All bits set to 1" << std::endl;
-
```

Comment:
The code at https://github.com/cms-sw/cmssw/blob/1e22f188f24910bc4a5421ef143d256a46f9ac2c/L1Trigger/DTTriggerServerPhi/src/DTTSCand.cc#L130-L147 is similar enough to assume that the suggested changes are correct.